frontend/pages/hosts/ManageHostsPage/HostTableConfig.tsx

```diff
@@ -285,20 +285,15 @@ const allHostTableHeaders: IHostTableColumnConfig[] = [
     accessor: "gigs_disk_space_available",
     id: "gigs_disk_space_available",
     Cell: (cellProps: IHostTableNumberCellProps) => {
-      const {
-        id,
-        platform,
-        percent_disk_space_available,
-      } = cellProps.row.original;
+      const { platform, percent_disk_space_available } = cellProps.row.original;
       if (platform === "chrome") {
         return NotSupported;
       }
       return (
         <DiskSpaceIndicator
-          baseClass="gigs_disk_space_available__cell"
+          inTableCell
```

Comment:
I'd either rename this to something more descriptive (like `showAsSpaceAvailable` or `shortText` or `variant="short"`) or make this a custom formatter function that takes the disk space available as an argument and returns whatever markup you want (i.e. ` {gigsDiskSpaceAvailable} GB available`)